### ISSUE TITLE
Can we integrate schedule free learning rate over epochs from Meta research into MLX fine tuning?

### ISSUE THREAD

**Issue by sajinpgupta** [ml-explore/mlx-examples#662]
https://github.com/facebookresearch/schedule_free

**Comment by awni** [comment]
Those look cool. If someone if interested in contributing I think it would be really useful to see how much they help out of the box with LoRA fine-tuning.  If it looks useful then we could include them in the example as a starting point.

**Comment by tmm1** [comment]
this is where an implementation would go presumably: https://github.com/ml-explore/mlx/blob/d40e76809f9ddbd32dcc86a34772392590a7eaca/python/mlx/optimizers/optimizers.py#L454
